contracts/src/StabilityPool.sol

```diff
@@ -632,52 +664,6 @@ contract StabilityPool is LiquityBase, IStabilityPool, IStabilityPoolEvents {
         return initialDeposit * (firstPortion + secondPortion) / snapshots.P / DECIMAL_PRECISION;
```

Comment:
We should also truncate this to `yieldGainsOwed`, if we want to be consistent. Plus, the invariant tests rely on `getDepositorYieldGainWithPending()` to predict the precise amount of yield that's going to be received by the depositor, so there are currently some failures due to the inconsistency between `getDepositorYieldGain()` & `getDepositorYieldGainWithPending()`.